packages/mattermost-redux/src/selectors/entities/channels.ts

```diff
@@ -89,8 +92,28 @@ export function getChannelsInTeam(state: GlobalState): RelationOneToMany<Team, C
     return state.entities.channels.channelsInTeam;
 }
 
-export function getChannelsInPolicy(state: GlobalState): IDMappedObjects<Channel> {
-    return state.entities.channels.channelsInPolicy;
+export function getChannelsInPolicy() {
+    return (createSelector(
+        getAllChannels,
+        (state: GlobalState, props: {policyId: string}) => getDataRetentionCustomPolicy(state, props.policyId),
+        (getAllChannels, policy) => {
+            if (!policy) {
+                return [];
+            }
+
+            const policyChannels: Channel[] = [];
+
+            Object.entries(getAllChannels).forEach((item: [string, Channel]) => {
+                const [, channel] = item;
```

Comment:
`,` is mistake?
```suggestion
                const [channel] = item;
```

Reply:
item is built with 2 array items, the first is a string (channel_id) and the second one is the channel object. Putting `[, channel]` means I haven't made a variable for the string array item. I only need the channel object